css/styles.css

```diff
@@ -1,19 +1,23 @@
-* { 
+body{ 
   box-sizing: border-box; 
 }  
 
+.container{
+  width:800px;
```

Comment:
@akallbold now just center the container so that it show in the middle of the page 🔨

Reply:
🔨